src/frontend/AppContainerWrapper.tsx

```diff
@@ -11,42 +11,39 @@ import {
   EDITING_SCREEN_NAMES,
   ERROR_STORE_KEY,
   IS_E2E,
+  NAV_STORE_KEY,
   NO_PRACTICE_BAR,
   TEMP_HIDE_PRACTICE_MODE_UI,
   URI_PREFIX,
 } from "./constants";
-import SettingsContext from "./context/SettingsContext";
 // import useProjectInviteListener from "./hooks/useProjectInviteListener";
 import bugsnag from "./lib/logger";
 import { PracticeMode } from "./sharedComponents/PracticeMode";
 import DefaultContainer from "./Navigation/DefaultContainer";
 import OnboardingContainer from "./Navigation/OnboardingContainer";
+import { devExperiments, featureFlagOn } from "./lib/DevExperiments";
 
 // Turn on logging if in debug mode
 if (__DEV__) debug.enable("*");
 const log = debug("mapeo:App");
 
-// WARNING: This needs to change if we change the navigation structure
-const getNavStoreKey = (includeOnboarding: boolean) =>
-  `@MapeoNavigation@${includeOnboarding ? 9 : 8}`;
-
-const createNavigationStatePersister = (includeOnboarding: boolean) => async (
+const createNavigationStatePersister = () => async (
   navState: NavigationState
 ) => {
+  if (featureFlagOn) return;
+
   try {
-    await AsyncStorage.setItem(
-      getNavStoreKey(includeOnboarding),
-      JSON.stringify(navState)
-    );
+    await AsyncStorage.setItem(NAV_STORE_KEY, JSON.stringify(navState));
   } catch (err) {
     log("Error saving navigation state", err);
   }
 };
 
-const loadSavedNavState = async (includeOnboarding: boolean) => {
+const loadSavedNavState = async () => {
   try {
+    if (featureFlagOn) return null;
```

Comment:
nit: move this outside the try statement since we don't want to potentially catch reference errors (i.e. `featureFlagOn` is not defined)